Remotion/Web/Core/WebStringType.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Remotion.Web
+{
+  /// <summary>
+  /// Defines the kind of a <see cref="WebString"/> which determines if it is encoded when rendered.
+  /// </summary>
+  public enum WebStringType
+  {
+    /// <summary>
+    /// The <see cref="WebString"/> contains unencoded text, which will be encoded before being rendered.
+    /// </summary>
+    Unencoded,
```

Comment:
`PlainText` feels better here. Thoughts?

Reply:
`PlainText` is better